src/components/ModalHeader/ModalHeader.jsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import styles from "./ModalHeader.module.scss";
+import { IconButton } from "components";
+import { CloseSmall } from "components/Icon/Icons";
+import Icon from "../Icon/Icon";
+
+const ModalHeader = ({
+  className,
+  title,
+  titleClassName,
+  description,
+  descriptionClassName,
+  icon,
+  iconSize,
+  iconClassName,
+  hideCloseButton,
+  closeButtonAriaLabel,
+  attr = {}
+}) => {
+  return (
+    <div className={cx(styles.mondayStyleModalHeader, className)}>
+      <p {...attr.title} className={cx(titleClassName, styles.title)}>
```

Comment:
What is `attr`? Why it's not in the propTypes? 
Why title comes from there and not from `title`?

Reply:
attr is an object with various aria related attributes a11ydialog provides. attr.title are the attributes it provides for title. But I checked again and only the id is actually dynamic so I refactored it and ModalHeader only get an id